### ISSUE TITLE
UI with Qt

### ISSUE THREAD

**Issue by kleuter** [veracrypt/VeraCrypt#169]
Any plans to re-write UI using Qt in order to have the same (and better) UI & code on all platforms?

**Comment by jleclanche** [comment]
God please yes.

**Comment by jleclanche** [comment]
ZuluCrypt has a qt ui, just for the record: https://github.com/mhogomchungu/zuluCrypt (but I'm not sure how good it is)

**Comment by DevPre** [comment]
BUMP! The current Veracrypt GUI is awful under linux. It needs to be rebuild with a more modern framework like Qt5.

**Comment by Vizitor0** [comment]
![vc1](https://user-images.githubusercontent.com/39656468/40579566-6e5e38f8-612a-11e8-95ec-ec073f63f7d6.png)
![vc2](https://user-images.githubusercontent.com/39656468/40579567-72ca3482-612a-11e8-9270-4fa8a1becd69.png)
![vc3](https://user-images.githubusercontent.com/39656468/40579568-75647838-612a-11e8-8cd7-54d5a8551f55.png)
This is from Arch Linux (gtk2 2.24.32, gtk3 3.22.30) and VeraCrypt 1.22 ; and it is impossible to use _Volume creation wizard_.
so I suppose Qt5 will be great.

**Comment by jleclanche** [comment]
¯\\\_(ツ)_/¯

**Comment by kleuter** [comment]
God bless Qt for VeraCrypt, thanks.

**Comment by alt3r-3go** [comment]
Well, VC uses wxWidgets, which is cross-platform already, I think the bigger problem here is that essentially that's not exactly shared between Linux/MacOS and Windows versions (AFAIU, from just relatively cursory look while debugging something), so instead of moving to Qt it would probably be more beneficial to just sort this out. Or at least sort this out *first*.

**Comment by kleuter** [comment]
We still need Qt.

**Comment by cypherbits** [comment]
I don't think we need Qt. It would mean more libraries and potential vulnerabilities.

**Comment by kleuter** [comment]
@cypherbits getting rid of legacy wxWidgets and replacing it with modern Qt will NOT add extra libraries

**Comment by ZinZingoo** [comment]
yes, we still ... request Qt (or just Qt with static build) ... in 2025
